### DISCUSSION TITLE
about result[Reimplementation]

### DISCUSSION THREAD

**Discussion by 1640477469** [open-mmlab/mmdetection#9440] [Q&A]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### 💬 Describe the reimplementation questions

I want to ask a question：I use my own dataset, using one GPU to train faster_rcnn, lr=0.0025, 180epoch, AP50 on the training set can only reach about 45%, why is it so low?

### Environment

no

### Expected results

no

### Additional information

change class to 5;
use coco datasets

**Comment by ZwwWayne** [comment]
The low AP can be caused by many reasons. I think the results is reasonable. To improve the performance, you can try larger batch size and stronger data augmentation.
